.github/workflows/check-available-os-versions.sh

```diff
@@ -81,11 +78,16 @@ fetch_os_versions() {
 check_os_versions() {
 	osname="$1"; shift
 	known="$@"
-	available="$(fetch_os_versions ${osname})"
-	[ "${known}" = "${available}" ] || {
+	found="$(fetch_os_versions ${osname})"
+	[ "${known}" = "${found}" ] || {
 		SURPRISE_COUNT=$(expr 1 + ${SURPRISE_COUNT})
-		echo "==> known to us: ${known}"
-		echo "==> now existing: ${available}"
+		versions_known="$(mktemp)"
+		versions_found="$(mktemp)"
+		trap "rm -f '${versions_known}' '${versions_found}'" 0 2 3 15
```

Comment:
You may use `INT TERM EXIT HUP` (assuming that is what 0 2 3 15 are for) but nothing against using direct signals values.